include/sys/socket.h

```diff
@@ -366,38 +371,74 @@ struct linger {
 
 struct sockaddr {
 	sa_family_t sa_family;
-	char sa_data[14];
+	char sa_data _Checked[14];
 };
 
 struct sockaddr_storage {
 	sa_family_t ss_family;
-	char __ss_padding[128-sizeof(long)-sizeof(sa_family_t)];
+	char __ss_padding _Checked[128-sizeof(long)-sizeof(sa_family_t)];
 	unsigned long __ss_align;
 };
 
 int socket (int, int, int);
 int socketpair (int, int, int, int [2]);
 
 int shutdown (int, int);
-
-int bind (int, const struct sockaddr *, socklen_t);
-int connect (int, const struct sockaddr *, socklen_t);
+int bind(int fd,
+	const struct sockaddr *addr : byte_count(len),
+	socklen_t len);
+int socketpair(int domain, int type, int protocol,
```

Comment:
Is this `socketpair` function different from the one declared on line 384?

Reply:
Thanks for the catch! No, they should be only one `socketpair`. I'll remove the one on line 384.